README.md

```diff
@@ -35,35 +35,54 @@ At a high level, the GEDI subsetting algorithm does the following:
 
 ## Algorithm Inputs
 
-To run a GEDI subsetting DPS job, you must supply the following inputs (a value
-must be supplied for every input):
+To run a GEDI subsetting DPS job, you must supply the following inputs.
 
-- `aoi`: URL to a GeoJSON file representing your area of interest (see
-  [Specifying an AOI](#specifying-an-aoi))
+|**IMPORTANT:**|
+|:---|
+| _When submitting a job via the ADE's user interface, to indicate that you do not want to specify a particular value for an **optional** input, you must enter a single dash/hyphen (`-`) in the input field.  This will indicate that you wish to use the default value indicated for that optional input._
 
-- `doi`: [Digital Object Identifier] (DOI) of the GEDI collection to subset, or
-  a logical name representing such a DOI (see
+- `aoi` (_required_): URL to a GeoJSON file representing your area of interest
+  (see [Specifying an AOI](#specifying-an-aoi)).  This may contain multiple
+  geometries.
+
+- `doi` (_required_): [Digital Object Identifier] (DOI) of the GEDI collection to
+  subset, or a logical name representing such a DOI (see
   [Specifying a DOI](#specifying-a-doi))
 
-- `lat`: Name of the dataset used for latitude.
+  Since 0.3.0
```

Comment:
I wonder if we could organize the inputs by release version similar to the CHANGELOG. It starts to get slightly confusing seeing it above/below another input and seeing it only some of the time

Reply:
Sure thing. Good suggestion! I plan to get this wrapped up this week. I keep getting interrupted, but I'm close.